### ISSUE TITLE
Failed chi2s evaluation

### ISSUE THREAD

**Issue by elbaraim** [PEtab-dev/PEtab#390]
I am trying to evaluate the chi2s for the Schwen_PONE2014 model in the Benchmark collection and I get the following error (files are attached down below):
``` python
data_df = pd.read_csv('measurementData_Schwen_PONE2014.tsv',sep='\t')
sim_df = pd.read_csv('simulatedData_Schwen_PONE2014.tsv',sep='\t')
obs_df = pd.read_csv('observables_Schwen_PONE2014.tsv',sep='\t')
par_df = pd.read_csv('parameters_Schwen_PONE2014.tsv',sep='\t')

petab.calculate.calculate_chi2(measurement_dfs=data_df,
                               simulation_dfs=sim_df,
                               observable_dfs=obs_df,
                               parameter_dfs=par_df)
```
``` python
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-34-a76910e46e25> in <module>
      2                                simulation_dfs=sim_df,
      3                                observable_dfs=problem.observable_df,
----> 4                                parameter_dfs=problem.parameter_df)

/usr/local/lib/python3.7/dist-packages/petab-0.1.4-py3.7.egg/petab/calculate.py in calculate_chi2(measurement_dfs, simulation_dfs, observable_dfs, parameter_dfs, normalize, scale)
    225     residual_dfs = calculate_residuals(
    226         measurement_dfs, simulation_dfs, observable_dfs, parameter_dfs,
--> 227         normalize, scale)
    228     chi2s = [calculate_chi2_for_table_from_residuals(df)
    229              for df in residual_dfs]

/usr/local/lib/python3.7/dist-packages/petab-0.1.4-py3.7.egg/petab/calculate.py in calculate_residuals(measurement_dfs, simulation_dfs, observable_dfs, parameter_dfs, normalize, scale)
     58         residual_df = calculate_residuals_for_table(
     59             measurement_df, simulation_df, observable_df, parameter_df,
---> 60             normalize, scale)
     61         residual_dfs.append(residual_df)
     62     return residual_dfs

/usr/local/lib/python3.7/dist-packages/petab-0.1.4-py3.7.egg/petab/calculate.py in calculate_residuals_for_table(measurement_df, simulation_df, observable_df, parameter_df, normalize, scale)
     95         mask = reduce(lambda x, y: x & y, masks)
     96 
---> 97         simulation = simulation_df.loc[mask][SIMULATION].iloc[0]
     98 
     99         if scale:

/usr/local/lib/python3.7/dist-packages/pandas-1.0.2-py3.7-linux-x86_64.egg/pandas/core/indexing.py in __getitem__(self, key)
   1766 
   1767             maybe_callable = com.apply_if_callable(key, self.obj)
-> 1768             return self._getitem_axis(maybe_callable, axis=axis)
   1769 
   1770     def _is_scalar_access(self, key: Tuple):

/usr/local/lib/python3.7/dist-packages/pandas-1.0.2-py3.7-linux-x86_64.egg/pandas/core/indexing.py in _getitem_axis(self, key, axis)
   2136 
   2137             # validate the location
-> 2138             self._validate_integer(key, axis)
   2139 
   2140             return self._get_loc(key, axis=axis)

/usr/local/lib/python3.7/dist-packages/pandas-1.0.2-py3.7-linux-x86_64.egg/pandas/core/indexing.py in _validate_integer(self, key, axis)
   2061         len_axis = len(self.obj._get_axis(axis))
   2062         if key >= len_axis or key < -len_axis:
-> 2063             raise IndexError("single positional indexer is out-of-bounds")
   2064 
   2065     def _getitem_tuple(self, tup: Tuple):

IndexError: single positional indexer is out-of-bounds
```
- Please find attached the corresponding tsv files as zip --> [files.zip](https://github.com/PEtab-dev/PEtab/files/4348183/files.zip)

**Comment by yannikschaelte** [comment]
@elbaraim the problem was that there are empty columns in the dataframes. this has been addressed now in #392, so it should work after that. in addition note that two of your dataframes require to have ids set,

```python
obs_df = pd.read_csv('observables_Schwen_PONE2014.tsv',sep='\t').set_index('observableId')
par_df = pd.read_csv('parameters_Schwen_PONE2014.tsv',sep='\t').set_index('parameterId')
```

**Comment by dweindl** [comment]
> obs_df = pd.read_csv('observables_Schwen_PONE2014.tsv',sep='\t').set_index('observableId') par_df = pd.read_csv('parameters_Schwen_PONE2014.tsv',sep='\t').set_index('parameterId')

Recommended: `petab.get_{parameter,observable}_df`
